pandas/core/frame.py

```diff
@@ -9928,6 +9928,13 @@ def _reduce(
         assert filter_type is None or filter_type == "bool", filter_type
         out_dtype = "bool" if filter_type == "bool" else None
 
+        def _maybe_swap_axis(df, axis):
```

Comment:
why does this need to be refactored out from where it is now?

Reply:
changed it back to the original form. In some intermittent state I used the code in this function multiple times and therefore thought it to be better style to refactor it to a function, but it's not needed anymore